src/test/java/org/isf/medicalstock/Tests.java

```diff
@@ -152,12 +153,6 @@ static void setUpClass() {
 		testSupplier = new TestSupplier();
 		testMedicalStock = new TestMedicalStock();
 	}
-
-	@BeforeEach
-	void setUp() throws OHException {
-		cleanH2InMemoryDb();
-	}
-
 	@AfterAll
```

Comment:
Why did you remove the blank line before this method?

Reply:
That may be due to the ide reformat. Just fixed it.